tests/sentry/api/endpoints/test_organization_details.py

```diff
@@ -48,7 +54,7 @@ def test_simple(self):
         )
         # TODO(dcramer): we need to pare this down -- lots of duplicate queries
         # for membership data
-        with self.assertNumQueries(28, using='default'):
+        with self.assertNumQueries(33, using='default'):
```

Comment:
Not jazzed about this. I'd like to revisit this and see if I can shave a few queries off.